pages.fr/common/7z.md

```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> Un archiveur de fichiers avec un haut taux de compression.
+> Plus d'informations : <https://www.7-zip.org/>.
```

Comment:
Is the space before the colon necessary in French? It would be more consistent with other pages and translations if we removed it. Same goes for the other pages.

Reply:
This is actually the correct way to write in French. Characters like colon, semicolon, question mark and exclamation mark are supposed to be always preceded by a non-breakable space.

Source: https://en.wikipedia.org/wiki/Punctuation#Other_languages

However, on a day to day basis, many, it's not that rare to forget this rule.

Reply:
Cool - just checking :smiley_cat:  As you can tell, I'm don't speak French very well at all :P

(the last time I studied it was at GCSE)

Reply:
Oh man, this is truly interesting. Indeed a space before and after the colon should be used in French: http://jp.barralis.com/linux-man/man1/as.1.php